core/src/types/address.rs

```diff
@@ -658,7 +658,7 @@ pub fn tokens() -> HashMap<Address, (&'static str, Denomination)> {
 /// Temporary helper for testing, a hash map of tokens addresses with their
 /// MASP XAN incentive schedules. If the reward is (a, b) then a rewarded tokens
 /// are dispensed for every b possessed tokens.
-pub fn masp_rewards() -> HashMap<Address, (u64, u64)> {
+pub fn masp_rewards() -> HashMap<Address, (u32, u32)> {
```

Comment:
Was this a necessary change? Neither type can be safely cast to `i32`. Of course, we know all these values fit into `i32`.